jbpm-flow-builder/src/main/java/org/jbpm/osgi/flow/compiler/Activator.java

```diff
@@ -35,7 +36,8 @@ public void start(BundleContext bc) throws Exception {
         this.processBuilderReg = bc.registerService( new String[]{ ProcessBuilderFactoryService.class.getName(), Service.class.getName()},
                                                                    new ProcessBuilderFactoryServiceImpl(),
                                                                    new Hashtable() );
-        ProcessBuilderFactory.reInitializeProvider();
+        RuntimeManagerFactory.Factory.reset();
+        ProcessBuilderFactory.reInitializeProvider();        
```

Comment:
we could and actually that was my first attempt though when jbpm-flow or jbpm-flow-builder is started there might still not be all stuff available - in this case jbpm-runtime-manager so it just makes sure it resets and allows "clean start" for clients calls. While the other makes sense to eagerly init as it's possible.